packages/client-sdk-nodejs/test/unit/auth-client.test.ts

```diff
@@ -168,5 +166,290 @@ describe('internal auth client', () => {
         grpcPermissions
       );
     });
+
+    it('creates expected grpc permissions for write-only cache and topic permissions', () => {
```

Comment:
had missed the unit test files when adding WriteOnly roles so adding tests now to cover that as well